<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>403</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Anonymous</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 141, 141);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_num1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 220, 220);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 199, 199);
}
QPushButton:pressed {
	background-color: rgb(1, 220, 121);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(7, 255, 131);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(6, 212, 109);
}
QPushButton:pressed {
	background-color: rgb(0, 241, 241);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sub">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_div">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>63</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 254, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tan">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sqrt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 255, 255);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(0, 226, 226);
}
QPushButton:pressed {
	background-color: rgb(6, 248, 135);
}

</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(7, 255, 131);
	border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 0, 0) ;
	padding-top: 5px;
	color: rgb(0, 0, 0);
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(6, 212, 109);
}
QPushButton:pressed {
	background-color: rgb(0, 241, 241);
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
